mmv1/third_party/terraform/services/gemini/resource_gemini_release_channel_setting_test.go.tmpl

```diff
@@ -60,6 +61,7 @@ resource "google_gemini_release_channel_setting" "example" {
     provider = google-beta
     release_channel_setting_id = "%{setting_id}"
     location = "global"
+    labels = {"my_key": "my_value"}
```

Comment:
per https://developer.hashicorp.com/terraform/language/expressions/types#map these should all use `=` rather than `:` - I'm actually surprised this works, but it would still be better to match standard.

```suggestion
    labels = {"my_key" = "my_value"}
```